source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
 
-.. localizedfigure:: images/geographic_search__explore.png
+To use the selectable filters:
 
-   The mini-map used for filtering on the catalog page.
+1. Choose a category of filters.
+2. Under the name of the category of filters are listed values (which are the metadata set for the datasets). Click one of the values. The other values disappear and the catalog directly updates to only display datasets matching the newly applied filter.
 
-Click on the little rectangle button on the top left of the map to select the drawing tool, and draw a rectangle on the area you want on the map to activate the filter. To cancel a selection, just click on the rectangle button again.
+.. admonition:: Note
+   :class: note
 
-This feature can be enabled in the back office, in the ``Configuring`` section, under the ``Configure catalog & dataset pages`` link. See :doc:`Configure catalog & dataset pages </configuring_domain/05_configuring_domain/tabs>` for more details.
+   For some categories, it can be possible to select more than one value. In this case, after clicking on a value, the others do not disappear. Click on any other value to filter even more.
 
-Filters
-~~~~~~~
+   For other categories, such as "Modification", the values are sorted in a hierarchical way. It means that the year must be selected first, then the months are displayed. Once the month is selected, the days are displayed in order to be selected as well.
 
-Under the geographic search map, there is a list of selectable filters sorted by category. They are constructed from the datasets metadata and enable the user to filter all the datasets that have the same attributes like language, theme, keywords, publisher, date, among others.
+Downloading the catalog
+-----------------------
 
-.. image:: images/filters_facets.png
-   :alt: Selectable filters
+Under all the filters are different possibilities to download the catalog. This does not allow to download all of the datasets from the catalog (or the filtered ones), but it allows to download an index of the datasets from the catalog along with their main metadata.
 
-Click on the values you want to use to filter the dataset list. For some categories
-like keywords and themes, it is possible to select multiple values, and for some
-others like modification date, the selection is hierarchical. For example,
-selecting a year will let you to choose the month.
+.. image:: images/download_catalog.png
+   :alt: Possible formats to download the catalog
 
-In the case of keywords, you can also click on them directly on a dataset card to
-filter the catalog on that keyword.
+It is possible to download the catalog in the following formats:
+
+- XLS
+- CSV
+- RDF (Turtle)
+- RDF (XML)
+- RSS feed
```

Comment:
It may be worth mentioning that using the API, they can download it in JSON, data.json, and possibly DCAT-AP
It's not displayed on this menu though.